extensions/script/python/PyProcCreator.h

```diff
@@ -56,6 +58,7 @@ class PythonObjectFactory : public core::DefautObjectFactory<minifi::python::pro
   /**
    * Create a shared pointer to a new processor.
    */
+
```

Comment:
very minor, but why did you add this empty line?

I would remove it, and I would also remove the comment, and all four doxygen comments from the `create()` functions, as they are 50% incorrect and 100% unnecessary